builder.config.json

```diff
@@ -200,9 +194,6 @@
           "primary": {
             "name": "Valid",
             "test": "@ISNUMBER(block.value)"
-          },
-          "default": {
-            "name": "Invalid"
```

Comment:
@object-required , As discussed, I'll put my review on hold until we have a clarification from Nancy what she mean about 
-`The final exit for every block type (in every mode) should be named “Default”`
- `This is true for most block right now, but should be updated for`
 (especially for Numeric block)

Reply:
This is resolved after our standup meeting